test/unit/get-config-test.js

```diff
@@ -673,6 +673,78 @@ describe('getProjectConfig', function () {
     let reprocessedConfig = getProjectConfig(project.baseDir, { config });
     expect(reprocessedConfig.rules).toEqual(expected);
   });
+
+  it('rules override extends', function () {
```

Comment:
Can you add another test like this one, that uses overrides by filePath? It **should** work the same with your fixes, but I want to make sure it stays under test too...

Reply:
Also lets avoid the word "override" here (since we have an actual concept in this domain that is `overrides`)

Reply:
Done. I added the new test cases in `modules-status-cache-test.js` because that's where the config tests for a specific file lives, but I had to add a `getProjectConfig()` first so that we cover all cases (`extends` vs `rules` vs `overrides`).